CHANGELOG.md

```diff
@@ -1,5 +1,95 @@
+- `kubernetes.io/tls` for TLS secrets.
+- `nginx.org/jwk` for JWK secrets.
+- `nginx.org/ca` for CA secrets.
+
+The Ingress Controller now ignores secrets that are not of a supported type. As a consequence, special upgrade steps are required.
+
+Before upgrading, ensure that the secrets referenced in Ingress, VirtualServer or Policies resources are of a supported type, which is configured via the `type` field. Because that field is immutable, it is necessary to either:
+* Recreate the secrets. Note that in this case, the client traffic for the affected resources will be rejected for the period during which a secret doesn't exist in the cluster.
+* Create copies of the secrets and update the affected resources to reference the copies. The copies need to be of a supported type. In contrast with the previous options, this will not make NGINX reject the client traffic.
+
+It is also necessary to update the default server secret and the wildcard secret (if it was configured) in case their type is not `kubernetes.io/tls`. The steps depend on how you installed the Ingress Controller: via manifests or Helm. Performing the steps will not lead to a disruption of the client traffic, as the Ingress Controller retains the default and wildcard secrets if they are removed.
+
+For *manifests installation*:
+1. Recreate the default server secret and the wildcard secret with the type `kubernetes.io/tls`.
+1. Upgrade the Ingress Controller.
+
+For *Helm installation*, there two cases:
+1. If Helm created the secrets (you configured `controller.defaultTLS.cert` and `controller.defaultTLS.key` for the default secret and `controller.wildcardTLS.cert` and `controller.wildcardTLS.key` for the wildcard secret), then no special upgrade steps are required: during the upgrade, the Helm will remove the existing default and wildcard secrets and create new ones with different names with the type `kubernetes.io/tls`.
+1.  If you created the secrets separately from Helm (you configured `controller.defaultTLS.secret` for the default secret and `controller.wildcardTLS.secret` for the wildcard secret):
+    1. Recreate the secrets with the type `kubernetes.io/tls`.
+    1. Upgrade to the new Helm release.
+
+NOTES:
+* Helm 2 clients are no longer supported due to reaching End of Life: https://helm.sh/blog/helm-2-becomes-unsupported/
+
+RESOURCES:
+
+* Documentation -- https://docs.nginx.com/nginx-ingress-controller/
+* Configuration examples -- https://github.com/nginxinc/kubernetes-ingress/tree/v1.10.0/examples and https://github.com/nginxinc/kubernetes-ingress/tree/v1.10.0/examples-of-custom-resources
+* Helm Chart -- https://github.com/nginxinc/kubernetes-ingress/tree/v1.10.0/deployments/helm-chart
+* Operator -- a compatible Operator version for this release will be published shortly.
+
```

Comment:
I know this is my fault as I drafted this content, but I should've mentioned. This content is only for the GitHub release. - I only included it in my draft to have it reviewed.

For example:
Note that is is present here: https://github.com/nginxinc/kubernetes-ingress/releases/tag/v1.9.0
But is omitted here: https://github.com/nginxinc/kubernetes-ingress/blob/master/CHANGELOG.md#190